tests/expectations/core/test_expect_column_values_to_be_of_type.py

```diff
@@ -1,10 +1,48 @@
+from unittest.mock import Mock
+
 import pandas as pd
 import pytest
+from snowflake.sqlalchemy import (
```

Comment:
I believe you should use the `great_expectations/compatibility/snowflake.py` here.